src/components/footer.html

```
@@ -1,4 +1,4 @@
-<footer class="w-full premium-footer pb-6 pt-12 px-6 sm:px-8">
+<footer class="w-full premium-footer pb-32 sm:pb-6 pt-12 px-6 sm:px-8">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)